presto-hive/src/test/java/com/facebook/presto/hive/TestHiveUtil.java

```diff
@@ -87,6 +94,20 @@ public void testToPartitionValues()
         assertToPartitionValues("pk=__HIVE_DEFAULT_PARTITION__");
     }
 
+    @Test
+    public void testShouldUseRecordReaderFromInputFormatTrue()
+    {
+        StorageFormat storageFormat = StorageFormat.create("parquet.hive.serde.ParquetHiveSerDe", "org.apache.hudi.hadoop.realtime.HoodieParquetRealtimeInputFormat", "");
+        assertTrue(shouldUseRecordReaderFromInputFormat(new Configuration(), new Storage(storageFormat, "test", Optional.empty(), true, Collections.emptyMap(), Collections.emptyMap())));
+    }
+
+    @Test
+    public void testShouldUseRecordReaderFromInputFormatFalse()
+    {
+        StorageFormat storageFormat = StorageFormat.create("parquet.hive.serde.ParquetHiveSerDe", "org.apache.hudi.hadoop.HoodieParquetInputFormat", "");
```

Comment:
Let's merge it into a one test case method. It generally verifies the same thing, but just different return